doc/nrf/releases/release-notes-latest.rst

```diff
@@ -30,6 +30,9 @@ There are no entries for this section yet.
 nRF9160
 =======
 
+* Added:
+  * :ref:`lib_modem_jwt` library, which provides an API get a JSON Web Token (JWT) from the modem.  Functionality requires modem firmware 1.3 or greater.
```

Comment:
What happens if the device has an older mfw?  Is this detected and reported at runtime, or if not, do suitable errors occur and get logged?

Reply:
the at cmd will fail and `modem_jwt_generate` will return `-EBADMSG`.
i am not aware of the standard way to handle such an issue.